src/styles/app.scss

```diff
@@ -25,6 +25,10 @@ body {
   background-color: var(--black);
 }
 
+iframe {
```

Comment:
Oh probably don't need this anymore. Should I remove?

Reply:
Width looks like it's already 100%